llmfoundry/utils/model_download_utils.py

```diff
@@ -94,11 +102,14 @@ def download_from_hf_hub(
             f'No supported model weights found in repo {model}.' +
             ' Please make sure the repo contains either safetensors or pytorch weights.'
         )
+    
+    allow_patterns = TOKENIZER_FILES if tokenizers_only else None
```

Comment:
```suggestion
    allow_patterns = TOKENIZER_FILES if tokenizer_only else None
```
For consistency with above suggestion